docs/contributing/architecture.md

```diff
@@ -1,17 +1,18 @@
----
-layout: page
-title: Architecture
-permalink: /architecture/
-nav_order: 1
-parent: Contributing
----
+--------------------------------------------------------------------------------
+
+layout: page title: Architecture permalink: /architecture/ nav_order: 1
```

Comment:
Probably caused by mdformat, but this looks incorrect, each option should be on a new line.

Reply:
Oh wow, thanks for flagging this, I'd missed it. I think I'll turn off `mdformat` in my `.vimrc` given it seems very determined to mutilate this.

Fixed in d9ee501